docs/Developer-guide/Development-Workflow.md

```diff
@@ -337,8 +350,9 @@ necessary condition for merge along with code review points.
 
 To run all regession tests locally, run below script from glusterfs root directory.
 
-  sh$ ./run-tests.sh
-
+```console
```

Comment:
Do we have to document about installing perl harness as thats a requirement to run prove commands?

Reply:
same as above. Content improvements can be different PR